salt/pillar/git_pillar.py

```diff
@@ -518,7 +518,7 @@ def __virtual__():
                     'could not be loaded, is GitPython installed?'
                 )
                 return False
-            if not git.__version__ > '0.3.0':
+            if git.__version__ <= '0.3.0':
```

Comment:
I know it's arguably unrelated to this PR, but... do we not have a better way to check version strings? I don't know if git guarantees it's version #s will be 0-9, and only 3 segments, but this seems mildly sketchy to me.